aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -653,10 +657,17 @@ public void onError(Exception exception) {
     @Override
     public void resetPassword(
             @NonNull String username,
+            @NonNull AuthResetPasswordOptions options,
```

Comment:
It would be great if overriding is used instead of this breaking change.

Reply:
`resetPassword()` on line 698 has the same method signature as the existing method. This will not be breaking.